compoments/report-repair/address.js

```diff
@@ -48,6 +48,9 @@ const Address = ({handleChange, values}) => {
   }
 
   return <div className="govuk-grid-row" data-cy="address">
+    <header>
+      <title>Select an address- Housing repairs</title>
```

Comment:
Please add a space between the page title and service title, i.e. 
```suggestion
      <title>Select an address - Housing repairs</title>
```

This is relevant to all of the changed components.